backend/proxy/Makefile

```diff
@@ -1,2 +1,2 @@
 all:
-	pushd ../.. && docker build -t vertaaiofficial/modeldb-proxy -f backend/proxy/Dockerfile . && popd
+	cd ../..; docker build -t vertaaiofficial/modeldb-proxy -f backend/proxy/Dockerfile .
```

Comment:
there should be a `cd backend/proxy` at the end to have the same effect as popd

Reply:
fixed